.github/workflows/ci.yaml

```diff
@@ -43,6 +43,12 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - run: |
+          # Workaround for https://github.com/canonical/charmcraft/issues/1389#issuecomment-1880921728
+          pipx install tox
+          pipx install poetry
+          tox run -e build-wrapper
+          mv requirements-last-build.txt requirements.txt
```

Comment:
Q: why it is not a `canonical/data-platform-workflows/.github/workflows/check_libs_deps.yaml` ? :-)